### ISSUE TITLE
Output prose format other than documented

### ISSUE THREAD

**Issue by e2fletcher** [palantir/tslint#4515]
### Bug Report

- __TSLint version__: 5.12.1
- __TypeScript version__: 3.3.3
- __Running TSLint via__: CLI

Apparently the output format has changed.
Is it intentional?
If so, it has not been changed in the documentation.
https://palantir.github.io/tslint/formatters/prose/

with `tslint.json` configuration:

```json
{
    "defaultSeverity": "warning",
    "extends": [
        "tslint:recommended"
    ],
    "jsRules": {},
    "rules": {},
    "rulesDirectory": []
}
```
#### Actual behavior
```bash
➜  test-tslint tslint --project tsconfig.json -t prose

WARNING: /home/ender/workspace/test-tslint/test.ts:3:13 - Missing semicolon
WARNING: /home/ender/workspace/test-tslint/test.ts:5:1 - Calls to 'console.log' are not allowed.
```

#### Expected behavior
```bash
➜  test-tslint tslint --project tsconfig.json -t prose
WARNING: /home/ender/workspace/test-tslint/test.ts[3,13]: Missing semicolon
WARNING: /home/ender/workspace/test-tslint/test.ts[5,1]: Calls to 'console.log' are not allowed.
```

**Comment by JoshuaKGoldberg** [comment]
Yup, this was changed in #4222 following discussion in #3861. This better aligns with the output of TypeScript and common test formats.

It looks like that page has the right format documented?

> The default formatter which outputs simple human-readable messages.

> Sample Output
> ```
> ERROR: myFile.ts[1, 14]: Missing semicolon
> ```

**Comment by blueyed** [comment]
> It looks like that page has the right format documented?

No.

It should say `ERROR: myFile.ts:1:14 - Missing semicolon` there.

**Comment by JoshuaKGoldberg** [comment]
Oh, yes, you're right 😄 this should be changed.

**Comment by blueyed** [comment]
Thanks.

Can you provide input on if there is a ERROR/WARNING prefix always?
It looks like there is only "error" and "warning" (and "off") for severity, isn't there?
(https://github.com/neomake/neomake/pull/2313/files#diff-fcb968db903a53dffeee05395d35d327R30)

**Comment by JoshuaKGoldberg** [comment]
Correct: https://github.com/palantir/tslint/blob/28ae0c1e6b3e1c578d5d5a326a94cdf3f1e5eb6d/src/language/rule/rule.ts#L101
